src/current/_includes/cockroachcloud/download-the-binary.md

```diff
@@ -1,21 +1,29 @@
     <section class="filter-content" markdown="1" data-scope="mac">
-    If you have not done so already, run the first command in the dialog to install the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
+    If you have not done so already, run the first command in the dialog to install the latest downloadable version of the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach /usr/local/bin/
+    curl https://get.cockroachdb.com/dl/cockroach-latest.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-latest.darwin-10.9-amd64/cockroach /usr/local/bin/
```

Comment:
The exact command is

```
curl -L https://get.cockroachdb.com/dl/cockroach-latest.darwin-10.9-amd64.tgz | tar -xz; sudo cp -i cockroach-*/cockroach /usr/local/bin/
```

![image](https://github.com/cockroachdb/docs/assets/3051672/0ba789c4-38f5-41dd-a853-90e56319e858)

(potentially for another day/out of scope here, but the Serverless instructions deal with this by not duplicating the command in the docs, and instead leaving the steps pretty high-level:
https://www.cockroachlabs.com/docs/cockroachcloud/connect-to-a-serverless-cluster#connect-to-your-cluster)

Reply:
Yes, I think bringing Dedicated and Serverless content into sync is out of sync for this effort.

Aha, I missed the `-L`, which is needed for `curl` to follow redirects.

Reply:
Now rewritten to avoid providing the curl commands directly in docs.